CMakeLists.txt

```diff
@@ -39,6 +39,15 @@ else()
   set (EXTRA_TEST_LIB_DEPS)
 endif()
 
+set(skip_pybind11_default_value OFF)
+if (MSVC)
```

Comment:
We can probably comment here that we are disabling them by default because we don't have active CI on them but they could potentially work.